src/environment.js

```diff
@@ -0,0 +1,20 @@
+const PRODUCTION = 'production'
+
+export const Env = {
+  isProduction: () => process.env.REACT_APP_ENV === PRODUCTION,
+
+  TILE_KEY: 'TILE_KEY',
+
+  getEnv: service => {
+    const envs = {
+      production: {
+        TILE_KEY: process.env.TILE_KEY
```

Comment:
Isso vai funcionar quando for criado a partir de um `create-react-app`? Eu lembro de ter lido que nesse caso sempre precisa ter REACT_APP no nome da variável

Reply:
Isso funciona em tempo de execução, já no bundle é o valor de fato que é printado aí.